pyccl/nl_pt/bacco_lbias.py

```diff
@@ -0,0 +1,447 @@
+__all__ = ("BaccoLbiasCalculator",)
+
+import numpy as np
+
+from .. import (CCLAutoRepr, CCLError, Pk2D,
+                get_pk_spline_a, unlock_instance)
+
+
+# All valid Pk pair labels and their aliases
+_PK_ALIAS = {
+    'm:m': 'm:m', 'm:b1': 'm:b1', 'm:b2': 'm:b2',
+    'm:b3nl': 'zero', 'm:bs': 'm:bs', 'm:bk2': 'm:bk2',
+    'b1:b1': 'b1:b1', 'b1:b2': 'b1:b2', 'b1:b3nl': 'zero',
+    'b1:bs': 'b1:bs', 'b1:bk2': 'b1:bk2', 'b2:b2': 'b2:b2',
+    'b2:b3nl': 'zero', 'b2:bs': 'b2:bs', 'b2:bk2': 'b2:bk2',
+    'b3nl:b3nl': 'zero', 'b3nl:bs': 'zero',
+    'b3nl:bk2': 'zero', 'bs:bs': 'bs:bs',
+    'bs:bk2': 'bs:bk2', 'bk2:bk2': 'bk2:bk2'}
```

Comment:
Just a comment here: at some point we may want to consider making _PK_ALIAS common to the whole `nl_pt` module to avoid repeating it in different submodules. Can you just add a `TODO` comment here?

Reply:
yep, will add!